### PR TITLE
Minor changes and bug fixes

### PR DESCRIPTION
**Pull request by maturanomx**
My PR is a:

- 🐛 Bug fix
- 💅 Enhancement

Main update on the:

Framework

Just some minor changes and bug fixes that I come across looking around the code. I put all along hoping should not take too much time to review them.

### CODE REVIEW

File: packages/strapi-admin/scripts/preSetup.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-shell.echo('🕓  The setup process can take few minutes.');
-shell.echo('');
-shell.echo('🔸  Administration Panel');
-shell.echo('📦  Installing packages...');
+shell.echo(`
+🕓  The setup process can take few minutes.
+
+🔸  Administration Panel
+📦  Installing packages...
+`);
```
Comment (maturanomx) [inline packages/strapi-admin/scripts/preSetup.js]: Just one call... 🤷‍♂️

File: packages/strapi-helper-plugin/lib/internals/webpack/webpack.dll.babel.js [hunk 2]
```diff
@@ -1,26 +1,17 @@
...
-const appPath = (() => {
-  if (process.env.APP_PATH) {
-    return process.env.APP_PATH;
-  }
-
-  return isAdmin ? path.resolve(process.env.PWD, '..') : path.resolve(process.env.PWD, '..', '..');
-})();
 // const isSetup = path.resolve(process.env.PWD, '..', '..') === path.resolve(process.env.INIT_CWD);
 const isSetup = process.env.IS_MONOREPO;
+const appPath = process.env.APP_PATH || path.resolve(process.env.PWD, '..', ( isAdmin ? '' : '..' ));
```
Comment (maturanomx) [inline packages/strapi-helper-plugin/lib/internals/webpack/webpack.dll.babel.js]: 6 lines reduced in just one

File: packages/strapi-plugin-content-manager/admin/src/components/TableRow/index.js [hunk 3]
```diff
@@ -74,7 +74,7 @@ class TableRow extends React.Component {
     <td key='action' className={styles.actions}>
       <IcoContainer
         icons={[
-          { icoType: 'pencil', onClick: () => this.handleClick(this.props.destination) },
+          { icoType: 'pencil', onClick: this.handleClick },
```
Comment (maturanomx) [inline packages/strapi-plugin-content-manager/admin/src/components/TableRow/index.js]: `handleClick` defined on line 69 is not accepting parameters. Definition and invocation are not the same.

This should be the more "difficult" to review

File: packages/strapi-plugin-content-manager/admin/src/components/TableRow/index.js [hunk 4]
```diff
@@ -119,7 +119,7 @@ class TableRow extends React.Component {
 
   render() {
     return (
-      <tr className={cn(styles.tableRow, this.props.enableBulkActions && styles.tableRowWithBulk)} onClick={() => this.handleClick(this.props.destination)}>
+      <tr className={cn(styles.tableRow, this.props.enableBulkActions && styles.tableRowWithBulk)} onClick={this.handleClick}>
```
Comment (maturanomx) [inline packages/strapi-plugin-content-manager/admin/src/components/TableRow/index.js]: same

File: packages/strapi-plugin-content-manager/admin/src/components/TableRow/index.js [hunk 5]
```diff
@@ -147,7 +142,9 @@ TableRow.propTypes = {
 };
 
 TableRow.defaultProps = {
+  enableBulkActions: true,
   onDelete: () => {},
+  value: false,
```
Comment (maturanomx) [inline packages/strapi-plugin-content-manager/admin/src/components/TableRow/index.js]: There was two `TableRow.defaultProps` defintions. I merged in one either the original code is one replacing the other... isn't?.

Let me know if I should just remove the first definition.

File: packages/strapi-plugin-users-permissions/admin/src/components/List/index.js [hunk 6]
```diff
@@ -38,7 +38,7 @@ const generateListTitle = (data, settingType) => {
 
       const disabledProviders = size(data) - enabledProvidersSize > 1 ?
         <FormattedMessage id="users-permissions.List.title.providers.disabled.plural" values={{ number: size(data) - enabledProvidersSize }} />
-        : <FormattedMessage id="users-permissions.List.title.providers.disabled.plural" values={{ number: size(data) - enabledProvidersSize }} />;
+        : <FormattedMessage id="users-permissions.List.title.providers.disabled.singular" values={{ number: size(data) - enabledProvidersSize }} />;
```
Comment (maturanomx) [inline packages/strapi-plugin-users-permissions/admin/src/components/List/index.js]: Conditional was returning the same in both cases. I think this is the correct.

File: packages/strapi-admin/scripts/preSetup.js [hunk 7]
```diff
@@ -1,12 +1,12 @@
 const shell = require('shelljs');
 const path = require('path');
-const _ = require('lodash');
```
Comment (maturanomx) [inline packages/strapi-admin/scripts/preSetup.js]: Not used in the file

**Comment by lauriejim** [comment]
Thank you for this PR, just let me take a look :)

**Comment by lauriejim** [comment]
It works well, just want @soupette validation before merging.

File: packages/strapi-plugin-users-permissions/admin/src/components/List/index.js (hunk 6 above)
Comment (soupette) [reply packages/strapi-plugin-users-permissions/admin/src/components/List/index.js]: Good catch

**Pull request merged**